SQLLexer.g4

```diff
@@ -126,11 +123,28 @@ IDENTIFIER:
     | [A-Z_] [A-Z_0-9]* // Ordinary identifiers
 ; // TODO check: needs more chars in set
 
-NUMERIC_LITERAL: ((DIGIT+ ('.' DIGIT*)?) | ('.' DIGIT+)) ('E' [-+]? DIGIT+)? | '0x' HEX_DIGIT+;
+// literals
 
-BIND_PARAMETER: [@$] IDENTIFIER;
+BOOLEAN_LITERAL:
+    'true'
+    | 'false'
+;
+
+NUMERIC_LITERAL:
+    [0-9]+
+;
+
+BLOB_LITERAL:
+    '0x' [0-9a-f]+
```

Comment:
I kind prefer to use fragment defined below: so `'0x' HEX_DIGIT+`, same to the NUMERIC_LITERAL

Reply:
Ok

Reply:
Actually I sort've disagree. Hex digit is not used anywhere, it should just be removed IMO

Reply:
either is fun.  Maybe also delete NUMBERIC_LITERAL?

Reply:
We use numeric literal

Reply:
ah sorry, I mean `DIGIT`, you've deleted